src/ca65/instr.c

```diff
@@ -1538,3 +1707,270 @@ void HandleInstruction (unsigned Index)
     /* Call the handler */
     InsTab->Ins[Index].Emit (&InsTab->Ins[Index]);
 }
+
+/*//////////////////////////////////////////////////////////////////// */
+/* C39 Funcs */
+/*//////////////////////////////////////////////////////////////////// */
+
+static void C39PutArith (const InsDesc* Ins)
+{
+  EffAddr A;
+
+  /* Evaluate the addressing mode used */
+  if (EvalEA (Ins, &A))
+    {
+      if (A.AddrModeBit & AM65_DIR_IND_Y && CPU == CPU_C39_EMUL)
+        {
+          switch(Ins->BaseCode)
+            {
+              /* emulate (ind),y with jsb calls, the zp addr is put as a const */
+              /* which is fetched by the jsb func */
+            case 0xA0: 
+              Emit1(0x0B, A.Expr); /* LDA (zp),y -> JSB0 .db expr */
+              break;
+            case 0x80: 
+              Emit1(0x1B, A.Expr); /* STA (zp),y -> JSB1 .db expr */
+              break;
+            case 0x60: 
+              Emit1(0x2B, A.Expr); /* ADC (zp),y -> JSB2 .db expr */
+              break;
+            case 0xE0: 
+              Emit1(0x3B, A.Expr); /* SBC (zp),y -> JSB3 .db expr */
+              break;
+            case 0x20: 
+              Emit1(0x4B, A.Expr); /* AND (zp),y -> JSB4 .db expr */
+              break;
+            case 0x00: 
+              Emit1(0x5B, A.Expr); /* ORA (zp),y -> JSB5 .db expr */
+              break;
+            case 0x40: 
+              Emit1(0x6B, A.Expr); /* EOR (zp),y -> JSB6 .db expr */
+              break;
+            case 0xC0: 
+              Emit1(0x7B, A.Expr); /* CMP (zp),y -> JSB7 .db expr */
+              break;
+            default:
+              ErrorSkip ("ARITH: Invalid arith base code 0x%x", Ins->BaseCode);
```

Comment:
That seems like an internal ca65 error message.  And, there are other error messages like that one; I think that they would be seen only if there is a bug in ca65.  Therefore, they should be handled by calls to `Internal()`.